.github/workflows/ci.yaml

```diff
@@ -113,7 +113,7 @@ jobs:
           path: coverage
 
       - name: Format, sum & upload results to Code Climate
-        uses: paambaati/codeclimate-action@v2.7.4
+        uses: paambaati/codeclimate-action@v3.2.0
```

Comment:
Not sure why this is still failing but thank you for updating! Let's see if we can figure this out, otherwise I'll see about switching to Codecov which I have more experience with.

Reply:
I think this may be related to secrets and PR from forks, it is not the update itself. It is failing also with the very same version

Reply:
Yes, you're right. Sorry about that. I read your message in the middle of the migration and realized then that makes sense. Either way though, I've been committed to migrating to Codecov which is now ready.

Reply:
Ok the Codecov migration is merged so we can drop the this file change. Thank you again for your patience and help! Always appreciated.

Reply:
@javierjulio I will rebase #7851 so it can be merged.

As per this one, there are other issues to fix and I can't figure out the correct approach. I will try to suggest one. I think it will be acceptable since this only affects the CI

Reply:
Ok that sounds good to me. I appreciate your help. Thank you!